src/main/java/heronarts/lx/mixer/LXBus.java

```diff
@@ -333,10 +333,14 @@ public LXClip getClip(int index, boolean create) {
   }
 
   public LXClip addClip() {
-    return addClip(this.mutableClips.size());
+    return addClip(this.mutableClips.size(), false);
   }
 
   public LXClip addClip(int index) {
+    return addClip(index, false);
+  }
+
+  public LXClip addClip(int index, boolean loop) {
```

Comment:
This feels strange, I wouldn't start this pattern in general because components have loads of parameters. These parameters should be changed post-core API method and listeners should do the right side effects. Let's leave this file untouched.